src/app/modules/admin/components/profile-header/profile-header.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, OnDestroy, OnInit } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { combineLatest, Subject } from 'rxjs';
+import { AdminState } from '../../state/admin.state';
+import * as AdminSelectors from '../../state/admin.selectors'
+import { map } from 'rxjs/operators';
+
+@Component({
+    selector: 'app-profile-header',
+    templateUrl: './profile-header.component.html',
+    styleUrls: ['./profile-header.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ProfileHeaderComponent implements OnInit, OnDestroy {
+    name$ = this.store.pipe(select(AdminSelectors.selectName));
```

Comment:
readonly

Reply:
please do use takeUntill for every defined observable ` this.store.pipe(select(AdminSelectors.selectName), takeUntill(this.unsubscribe$));`